prow/cmd/checkconfig/main.go

```diff
@@ -1052,23 +1052,45 @@ func verifyOwnersPlugin(cfg *plugins.Configuration) error {
 	return nil
 }
 
-func verifyLabelPlugin(label plugins.Label) error {
+func checkRestrictedLabelNotInAdditionalLables(restricted string, label plugins.Label) bool {
+	for _, additional := range label.AdditionalLabels {
+		if restricted == additional {
+			return true
+		}
+	}
+	return false
+}
+
+func verifyLabelPlugin(label plugins.Label) []error {
```

Comment:
Let's make this function to return an actual error from the error list, using `https://pkg.go.dev/k8s.io/apimachinery/pkg/util/errors#NewAggregate`

Reply:
Done